scripts/update_libunwind.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+shopt -s globstar
+
+LIBUNWIND_FOLDER=$1
```

Comment:
Where can we find upstream sources for libunwind? If it's a git project, we might just clone it to some tmp directory in this script, this way updating in the future might be way easier. Otherwise, some links might be usefull.

Reply:
Yeah, it's just from the LLVM repo. I'll update the script to clone the sources into a temporary repo.

Reply:
Should be fixed now!